src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -490,3 +507,36 @@ def _json_column_op(self, this, path):
                 return self.expression(local_expression.Bracket, this=this, expressions=[path])
 
             return self.expression(exp.Bracket, this=this, expressions=[path])
+
+        def _parse_window(self, this: exp.Expression | None, alias: bool = False) -> exp.Expression | None:
+            window = super()._parse_window(this=this, alias=alias)
+            # Adding default window frame for the rank-related functions in snowflake
+            if (
+                window
+                and _contains_expression(
+                    window.this,
+                    (
```

Comment:
small nit:
Can you please create a set of these rank functions and pass the variable here, just to improve the readability?

Reply:
something like this ?
```suggestion
RANK_FUNCTIONS = {local_expression.CumeDist,
                        local_expression.DenseRank,
                        exp.FirstValue,
                        exp.Lag,
                        exp.LastValue,
                        exp.Lead,
                        local_expression.NthValue,
                        local_expression.Ntile,
                        local_expression.PercentRank,
                        local_expression.Rank,
                        exp.RowNumber,
}

_contains_expression(window.this, RANK_FUNCTIONS) 
```

Reply:
Done.